pkg/transaction/httpprotocolbuilder.go

```diff
@@ -375,9 +385,14 @@ func (b *httpProtocolBuilder) queryArgsRedaction() {
 	if b.err != nil {
 		return
 	}
-
+	var redactedArgs map[string][]string
```

Comment:
We should move setting the p.httpProtocol.URI in SetURI as well here under the Build to use the overridden redactionConfig instead.

Reply:
Fixed in the latest commit.